settings.py

```diff
@@ -968,6 +968,8 @@
 DATADOG_API_KEY = None
 DATADOG_APP_KEY = None
 
+TEST_SHOULD_USE_SQL_BACKEND = False
+
```

Comment:
I don't think this should be in the main settings file. If can go in travis settings at some point. Also here it is `TEST_` but  where it's referenced it is `TESTS_`